README.md

```diff
@@ -81,6 +81,49 @@ microovn.ovn-sbctl show
 microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluster/status OVN_Northbound
 ```
 
+## Downscaling
```

Comment:
I think we need to balance our documentation so that we won't end up copy-pasting upstream project documentation details. Perhaps rather than going into details about Raft and OVN resiliency, I could find relevant upstream documentation and link it?

Reply:
Sure, but I think it is important to highlight that two members with central components is worse than having one, so that people don't leave their clusters rubbing like that for extended periods of time.

Reply:
Yup, definitely +1. I'll try to find some good source on that to link. Or if I can't find any I'll just add the warning explicitly.